use rlang::is_call_simple (#746)
tidymodels/parsnip: NEWS.md

* Model type functions will now message informatively if a needed parsnip extension package is not loaded (#731).

> * Fixed bug where en error would be thrown if arguments to model functions were namespaced and find_tune_id() were used (#745).

Reviewer: ```suggestion
* Fixed a bug where an error would be thrown if arguments to model functions were namespaced (#745).
```

I don't think you need to mention the internal function in the news bullet. The PR link points to all the developer details
